### PR TITLE
Add Storage to SpotBugs Aggregate Report

### LINKED ISSUE #4925: Turn on SpotBugs Aggregate Report

### PR DESCRIPTION
**Pull request by alzimmermsft**
Fixes #4925 

Adds Storage modules to SpotBugs aggregate report and fixes any SpotBug issues reported.

### CODE REVIEW

File: sdk/storage/azure-storage-file/src/samples/java/com/azure/storage/file/FileSample.java [hunk 1]
```diff
@@ -90,8 +88,8 @@ public static void main(String[] args) throws Exception {
         }
 
         // Upload a local file to the storage.
-        URL fileFolder = FileSample.class.getClassLoader().getResource(".");
-        String uploadPath = fileFolder.getPath() + "testfiles/" + "uploadSample.txt";
+        String filePath = "C:/filePath/";
```
Comment (sima-zhu) [inline sdk/storage/azure-storage-file/src/samples/java/com/azure/storage/file/FileSample.java]: What's the reason change to C folder?

Comment (alzimmermsft) [reply sdk/storage/azure-storage-file/src/samples/java/com/azure/storage/file/FileSample.java]: Using the class loader get resource could lead to a NPE, given this is a sample not tying to anything real I just made it simpler.

**Pull request merged**